client/src/test/java/org/asynchttpclient/handler/resumable/ResumableAsyncHandlerTest.java

```diff
@@ -14,32 +14,183 @@
  */
 
 import static org.asynchttpclient.Dsl.get;
+import static org.mockito.Mockito.*;
+import static org.powermock.api.mockito.PowerMockito.mock;
```

Comment:
Where does the PowerMock dependency come from? I'm not opposed to it, but it should be declared explicitly in the parent pom along with the other test deps.